apps/nextra/pages/zh/_meta.ts

```diff
@@ -0,0 +1,53 @@
+export default {
+  index: {
+    title: "Introduction",
+    type: "page",
+    display: "hidden",
+    theme: {
+      layout: "raw",
+      footer: null,
+    },
+  },
+  learn: {
+    type: "page",
+    title: "Learn ↗",
+    href: "https://learn.aptoslabs.com/",
+    newWindow: true,
+  },
```

Comment:
This is out of date with the current structure

Reply:
> This is out of date with the current structure

Done